x/tunnel/README.md

```diff
@@ -131,22 +149,21 @@ type Packet struct {
 }
 ```
 
-#### Packet Generation
+#### Packet Generation Workflow
 
-At the end of every block, the tunnel generates packets by checking the deviations and intervals for each tunnel. We utilize both hard and soft deviations:
+At the end of each block, the tunnel generates packets by evaluating the deviations and intervals for all tunnels. The system uses two types of deviations: hard deviations and soft deviations, which are explained in detail below.
 
-- **Hard Deviation**: If any signal reaches this threshold, the system triggers a check on all soft deviations.
-- **Soft Deviation**: If any signal meets its soft deviation criteria during this check, the latest price is sent in the packet.
+If any signal exceeds the hard deviation threshold, it is appended to the list of signals to be sent. Additionally, if any signal meets its soft deviation criteria while another symbol surpasses the hard deviation threshold, that signal is also added to the list.
```

Comment:
"if any signal meets its soft deviation criteria while another symbol surpasses the hard deviation threshold, that signal is also added to the list."

Mixed use of signal and symbol
I don't know which one is 'that signal' maybe use signal a, signal b?

Reply:
But the first sentence is "If any signal exceeds the hard deviation threshold, it is appended to the list of signals to be sent" so `that` should be the `any signal meets its soft deviation criteria` because we already know that signal exceeds the the hard deviation threshold already append in the list.